social-guidelines.md

```diff
@@ -0,0 +1,53 @@
+* Informational: Tutorials, editorials, news stories. Items that help connect the dots for our community
+* Thought leadership: insights, perspectives, learnings, and experiences 
+
+## Strategy ##
+* Keep messages positive and uplifting, consistent with the [values and principles of the GraphQL Foundation](https://github.com/graphql/foundation/blob/master/CODE-OF-CONDUCT.md)
+* Communicate big picture ideas vs “announcements” or news. News will be positioned less like an announcement and more around what it means.
+* Share content across social media channels that benefit the ecosystem as a whole
+* Maintain a balance of posting on GraphQL Foundation and foundation members activities/news and ecosystem-focused content 
+* Share vendor-neutral, community-sourced posts that are informational, engaging, and ecosystem-focused
+* Engage with the community through retweets and sharing of community content.
+
+## General Guidelines ##
+
+For all GraphQL Foundation social activity, we remain a neutral foundation. Examples of the type of content:
+* Owned content is GraphQL Foundation news, blogs, case studies, survey data, etc.
+* Project content is anything sourced from [GraphQL Foundation’s currently hosted projects](https://github.com/graphql), including project news, roadmap updates, new releases, performance/security updates, blogs, conferences slides/videos, etc.
+* Ecosystem content is vendor-neutral and project-impartial sourced from contributors, maintainers, ambassadors, news outlets, etc., including blog posts, news coverage, thought leadership bylines, technology demos, sketch notes, GitHub work, GraphQL-specific Meetups, etc.
+  * CNCF is not able to share anything on our channels that promotes a vendor product or directs to a company website. 
+  * Social posts from member company handles and/or vendor channels cannot be shared.
+  * RTs are limited to news outlet handles, @graphql, project handles + personal handles.
+  * Community content that abides by the channel guidelines + includes insight from/work with more than one GraphQL Foundation project will be prioritized for sharing. 
+  * Activities that are hosted by and open to the public can be promoted, as they benefit the ecosystem as a whole. 
+
+## Images & Videos ##
+Images shared, unless specifically credited back to a community member, will meet the requirements of “free for commercial use” and “no attribution required.”
+
+## Hashtags ##
+GraphQL Foundation uses hashtags in our posts:
+* To measure the success of campaigns (for example, #GraphQL);
+* To expand our reach beyond our current followers and tap into larger, trending topics on Twitter (for example, #api)
+* To organize or categorize shares (for example, #GraphQL).
+
+## To request content to be shared ##
+Share your tweet on the [#socialmedia](https://app.slack.com/client/T08PSQ7BQ/C12MRQ97A) channel on the GraphQL Slack channel or email social@cncf.io.
```

Comment:
@brianwarner @caniszczyk we should open a channel for it and also update the email address?
Do we need an email address at all or we can simply just rely on Slack?

Reply:
@Urigo SGTM, it's up to @brianwarner but I would simply just reference the slack channel vs email